<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-RSB6NC4" failures="0" tests="4" name="extentSparkReportingV5.NopCommerceTest" time="0.259" errors="1" timestamp="2022-09-19T19:24:49 IST" skipped="3">
  <testcase classname="extentSparkReportingV5.NopCommerceTest" name="setupExtent" time="0.259">
    <error type="java.lang.ExceptionInInitializerError">
      <![CDATA[java.lang.ExceptionInInitializerError
at org.apache.log4j.Logger.getLogger(Logger.java:40)
at freemarker.log._Log4jLoggerFactory.getLogger(_Log4jLoggerFactory.java:31)
at freemarker.log.Logger.getLogger(Logger.java:362)
at freemarker.template.Configuration.<clinit>(Configuration.java:151)
at com.aventstack.extentreports.reporter.AbstractFileReporter.loadTemplateModel(AbstractFileReporter.java:54)
at com.aventstack.extentreports.reporter.ExtentSparkReporter.start(ExtentSparkReporter.java:136)
at com.aventstack.extentreports.reporter.ExtentSparkReporter.access$000(ExtentSparkReporter.java:27)
at com.aventstack.extentreports.reporter.ExtentSparkReporter$1.onSubscribe(ExtentSparkReporter.java:116)
at io.reactivex.rxjava3.subjects.PublishSubject.subscribeActual(PublishSubject.java:133)
at io.reactivex.rxjava3.core.Observable.subscribe(Observable.java:13099)
at com.aventstack.extentreports.ReactiveSubject.attachReporter(ReactiveSubject.java:38)
at com.aventstack.extentreports.ExtentReports.attachReporter(ExtentReports.java:66)
at extentSparkReportingV5.NopCommerceTest.setupExtent(NopCommerceTest.java:56)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:637)
at org.testng.TestRunner.beforeRun(TestRunner.java:627)
at org.testng.TestRunner.run(TestRunner.java:589)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: org.apache.logging.log4j.LoggingException: log4j-slf4j-impl cannot be present with log4j-to-slf4j
at org.apache.logging.slf4j.Log4jLoggerFactory.validateContext(Log4jLoggerFactory.java:71)
at org.apache.logging.slf4j.Log4jLoggerFactory.newLogger(Log4jLoggerFactory.java:51)
at org.apache.logging.slf4j.Log4jLoggerFactory.newLogger(Log4jLoggerFactory.java:33)
at org.apache.logging.log4j.spi.AbstractLoggerAdapter.getLogger(AbstractLoggerAdapter.java:53)
at org.apache.logging.slf4j.Log4jLoggerFactory.getLogger(Log4jLoggerFactory.java:33)
at org.slf4j.LoggerFactory.getLogger(LoggerFactory.java:358)
at org.apache.logging.slf4j.SLF4JLoggerContext.getLogger(SLF4JLoggerContext.java:36)
at org.apache.log4j.Category.<init>(Category.java:176)
at org.apache.log4j.Category.<init>(Category.java:192)
at org.apache.log4j.Logger.<init>(Logger.java:57)
at org.apache.log4j.spi.RootLogger.<init>(RootLogger.java:39)
at org.apache.log4j.LogManager.<clinit>(LogManager.java:72)
... 37 more
]]>
    </error>
  </testcase> <!-- setupExtent -->
  <system-out/>
  <testcase classname="extentSparkReportingV5.NopCommerceTest" name="nopCommerceLoginTest" time="0.000">
    <skipped/>
  </testcase> <!-- nopCommerceLoginTest -->
  <system-out/>
  <testcase classname="extentSparkReportingV5.NopCommerceTest" name="nopCommerceLogoTest" time="0.000">
    <skipped/>
  </testcase> <!-- nopCommerceLogoTest -->
  <system-out/>
  <testcase classname="extentSparkReportingV5.NopCommerceTest" name="nopCommerceTitleTest" time="0.000">
    <skipped/>
  </testcase> <!-- nopCommerceTitleTest -->
  <system-out/>
</testsuite> <!-- extentSparkReportingV5.NopCommerceTest -->
